pkgs/development/python-modules/flask-basicauth/default.nix

```
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, flask
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "Flask-BasicAuth";
+  version = "0.2.0";
```

The revision you specified isn't the v0.2.0 tag, if you'd still like to use this commit please use an unstable version:

```suggestion
  version = "unstable-2016-10-20";
```

---

my apologies, i intended to use the `v0.2.0` tag. i have updated `rev` and `sha256` accordingly.